src/Bootstrap/less/theme/page-display-package-v2.less

```
@@ -11,15 +11,63 @@
     border-bottom: 1px solid lightgray;
   }
 
+  .package-header {
+    margin-bottom: 36px;
+  }
+
   .package-title {
-    margin-bottom: 24px;
+    margin-bottom: 30px;
 
-    div {
-      margin-top: 2px;
+    h1 {
+
+      .title {
+        font-size: 32px;
+        margin-top: 40px;
+        font-weight: 400;
+        margin-right: 18px;
+      }
+
+      .version-title {
+        font-size: 16px;
+        font-weight: 400;
+        color: #666666;
```

I believe you can use the `@gray-light` variable here:

```suggestion
        color: @gray-light;
```

See:
https://github.com/NuGet/NuGetGallery/blob/db62de43e24b704e9ab16d8abc39bdfa3e73260b/src/Bootstrap/less/variables.less#L16

---

done 👍